x/mongo/driver/ocsp/ocsp.go

```diff
@@ -52,8 +52,28 @@ func newOCSPError(wrapped error) error {
 	return &Error{wrapped: wrapped}
 }
 
+// Response contains a subset of the details needed from an OCSP response after the original response has been
+// validated.
+type Response struct {
+	Status     int
+	NextUpdate time.Time
+}
+
+func newResponse(res *ocsp.Response) *Response {
```

Comment:
If you rename the type above, maybe call this `extractResponseDetails`.

Reply:
Done.